core/src/main/java/com/linecorp/armeria/client/retry/RetryingClient.java

```diff
@@ -32,6 +32,7 @@
  */
 public abstract class RetryingClient<I extends Request, O extends Response>
         extends SimpleDecoratingClient<I, O> {
+    private static final int DEFAULT_MAX_ATTEMPTS = 5;
```

Comment:
I was just curious that how we can define some hard-limit in other way instead of a constant. If it's not appropriate, I think the current is okay. Forget my previous comment. 😄